main/src/ca/uwaterloo/flix/util/Options.scala

```diff
@@ -175,3 +179,55 @@ object Subeffecting {
   case object Lambdas extends Subeffecting
 
 }
+
+sealed trait VerificationOptions {
+
+  /**
+    * Returns `true` if `verifier` is enabled by `this`.
+    */
+  def isEnabled(verifier: VerificationOptions.Verifier): Boolean = this match {
+    case VerificationOptions.EnableAll => true
+    case VerificationOptions.EnableSome(verifiers) => verifiers.contains(verifier)
+  }
+
+}
+
+object VerificationOptions {
+
+  /**
+    * Enable all verifiers.
+    */
+  case object EnableAll extends VerificationOptions
+
+  /**
+    * Enable the listed verifiers.
+    */
+  case class EnableSome(verifiers: Set[Verifier]) extends VerificationOptions
```

Comment:
@mlutze I kind of want to just have a cofinite set of Verifiers (easy to do all / all except X / just X) is that too weird you think? So the verifier option would just be `Coset[Verifier]`

Reply:
Then, below, I wouldn't have to define `Verifiers.all`

Reply:
Too weird.

Reply:
Yeah, oh well